packages/client/src/library/components/InputAddress.js

```diff
@@ -4,7 +4,14 @@ import { useAddressValidation } from "hooks";
 import Svg from "library/Svg";
 import { isAddr, formatAddress } from "utils";
 
-const InputAddress = ({ web3, value, isValid, onChange }) => {
+const InputAddress = ({ 
+  web3, 
+  value, 
+  isValid, 
+  onChange, 
+  className = "", 
+  readOnly = false,
```

Comment:
add `className` to add classes needed for situational positioning/styling and `readOnly` to render a disabled input